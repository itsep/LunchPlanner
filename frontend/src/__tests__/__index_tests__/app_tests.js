import React from 'react'
import renderer from 'react-test-renderer'
import App from '../../pages/index/components/app'

it('Homepage_app renders correctly', () => {
  fetch.once(JSON.stringify({ count: 0 }))
  const tree = renderer
    .create(<App />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
