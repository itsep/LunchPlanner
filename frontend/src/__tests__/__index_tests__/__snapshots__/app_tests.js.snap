// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage_app renders correctly 1`] = `
<h1>
  hi
</h1>
`;
